### ISSUE TITLE
Year Progression Graph / Presets doesn't show the current day delta

### ISSUE THREAD

**Issue by jcaude** [thomaschampagne/elevate#703]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions?
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues?

### Step 2: Describe your environment

  * Plugin version: 6.8.0
  * Chrome/Opera/Chromium version: 70.0
  * OS version: OS/X 10.14
  
### Step 3: Describe the problem:

When a preset is loaded in the YP graph, and the current day selected (manually or after a reset) the delta-target is updated for all selected years but not for the current one (aka 2018). Still, it works as expected for all other days.. 

#### Steps to reproduce:

  1. set a preset
  2. select any day but today and after
  3. select today or use reset
  
#### Actual Behavior:

  * Do you have javascript errors logs in browser console?

Nope

#### Media attachement:

AFTER STEP 2
<img width="1448" alt="capture d ecran 2018-11-22 a 17 20 01" src="https://user-images.githubusercontent.com/1268176/48914159-e8906c80-ee7a-11e8-8f93-0f98d953352b.png">

AFTER STEP 3 (see Table / 2018 / delta-target)
<img width="1451" alt="capture d ecran 2018-11-22 a 17 19 45" src="https://user-images.githubusercontent.com/1268176/48914185-f5ad5b80-ee7a-11e8-80d1-115e78fe1bde.png">

**Comment by thomaschampagne** [comment]
Ok i was able to reproduce. I will dive in it.

Thanks

**Comment by jcaude** [comment]
You're welcome.. and your are to be thanks given the amazing job you do. Keep going on..

**Comment by jcaude** [comment]
I look a little bit in your code, but I'm not that fluent in javascript, next time I will give it a try..

**Comment by thomaschampagne** [comment]
Do you still reproduce with that build: https://bitbucket.org/thomaschampagne/elevate-ci-builds/downloads/v6.8.0_stable_2018-11-23-11-58.develop.94d82978.zip

I'm not able to reproduce it anymore

(FYI This build fix also #702)

**Comment by jcaude** [comment]
I will check this evening.

**Comment by jcaude** [comment]
Tested ... it works as expected. Thx

**Comment by JamieCristofani** [comment]
This bug appears to have returned, at least prior to a run today.

![image](https://user-images.githubusercontent.com/38893912/69148309-e8e23880-0acb-11ea-9906-579336ab3b03.png)
